src/oneD/Domain1D.cpp

```diff
@@ -145,6 +145,31 @@ AnyMap Domain1D::serialize(const double* soln) const
     return out;
 }
 
+shared_ptr<SolutionArray> Domain1D::toArray(bool normalize) const {
+    if (!m_data) {
+        throw CanteraError("Domain1D::toArray",
+            "Domain needs to be installed in a container before calling asArray.");
+    }
+    if (!normalize) {
+        return asArray(m_data->data() + m_iloc);
+    }
+    auto ret = asArray(m_data->data() + m_iloc);
+    ret->normalize();
+    return ret;
```

Comment:
Very minor simplification:
```suggestion
    auto ret = asArray(m_data->data() + m_iloc);
    if (normalize) {
        ret->normalize();
    }
    return ret;
```